module.json

```
@@ -39,6 +39,8 @@
   "url": "This is auto replaced",
   "manifest": "This is auto replaced",
   "download": "This is auto replaced",
+  "license": "LICENSE",
+  "readme": "readme.md",
```

Should probably be capitalized ad the repo is initialized with capitalized, right?
```suggestion
  "readme": "README.md",
```
Needs to be adjusted in the workflow, too.

---

Could have sworn I checked that specifically... You're absolutely right